skorch/callbacks/scoring.py

```diff
@@ -100,6 +101,9 @@ def initialize(self):
     def on_train_begin(self, net, X, y, **kwargs):
         self.X_indexing_ = check_indexing(X)
         self.y_indexing_ = check_indexing(y)
+        with suppress(ValueError, IndexError):
+            idx_best = net.history[:, f'{self.name_}_best'].index(True)
```

Comment:
Is skorch 3.6-compatible only?

Reply:
`suppress` was introduced in python `3.4`: https://docs.python.org/3/whatsnew/3.4.html#contextlib

`skorch` only supports `3`, and the `3.X` versions `pytorch` supports are `3.5` and `3.6`, so I think `suppress` is okay to use.

Reply:
I was actually referring to the use of f-strings ;)

Reply:
OH I see! I updated this to use the `.format` style string instead.